admin/updates/20160621-non-mbid-submission.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+CREATE TYPE id_type AS ENUM ('mbid', 'msid');
+ALTER TABLE "lowlevel" RENAME COLUMN mbid TO gid;
+ALTER TABLE "lowlevel" ADD COLUMN is_mbid id_type;
```

Comment:
yes, `gid_type` is good. If it lets you, I would call the enum the same too. It's not worth calling it `gid_type_enum` because in this case we already know that it's an enum.